src/intl/en/page-upgrades.json

```diff
@@ -1,5 +1,8 @@
 {
   "page-upgrades-beacon-date": "Shipped!",
   "page-upgrades-merge-date": "~Q3/Q4 2022",
-  "page-upgrades-shards-date": "~2023"
+  "page-upgrades-shards-date": "~2023",
+  "page-upgrades-merge-banner-intro": "The Merge is approaching, and comes with changes to Ethereum.",
+  "page-upgrades-merge-banner-content-outdated": "Some content on this page may be out-of-date related to these changes, and updates are coming soon.",
+  "page-upgrades-merge-banner-developers-landing": "Some docs may be out-of-date related to these changes and have been marked with a banner similar to this. Updates coming soon."
```

Comment:
```suggestion
  "page-upgrades-merge-banner-developers-landing": "Some docs have a banner indicating they are out-of-date. Updates coming soon."
```

Reply:
This component is designed to have the same intro wherever its placed, and then the link at the end. 
The middle sentence is optional. It is absent on the homepage leaving the announcement to basically say the merge is coming, learn more. 
On the developers and docs landing pages the middle sentence denotes that some pages inside have outdated content, but those pages themselves are fine.
Then specific docs can be flagged for "preMerge Banner" and those are denoted as having some content that is out of date. I say "some" because these flagged pages are still mostly correct. 
We use the pattern of trailing links (ie. "More on ___") throughout the site.

Also didn't worry too much about line length given translations. We may not have everything translated in time for The Merge, but we still may want to consider translating this and using it until intl pages are up to date. 

Others can chime in, but that was the reasoning.